std-bits/table/src/main/java/org/enso/table/data/column/builder/LongBuilder.java

```diff
@@ -33,26 +45,72 @@ static LongBuilder make(int initialSize, IntegerType type, ProblemAggregator pro
     }
   }
 
+  static LongBuilder fromAddress(int size, long address, long validity, IntegerType type) {
+    assert address != 0;
+    var tripple = allocBuffer(size, address, validity);
+    var builder = new LongBuilder(tripple, null);
+    return builder;
+  }
+
+  /**
+   * Allocates continuous direct memory buffer. First of all there is a validity bit map (padded to
+   * 8 bytes) followed by the actual data.
+   *
+   * @param size the size of buffer to allocate
+   * @param data address of data to read or {@code 0} to allocate new data
+   * @param validity address of validity bitmap to read or {@code 0} to assume all data are valid
+   * @param initialSize the size of the buffer
+   * @return tripple of whole {@link ByteBuffer}, {@link BitSet} and {@link LongBuffer}
```

Comment:
Maybe I'm not seeing it in the diff `ByteBuffer` does not appear to be used anywhere so... drop it?